### DISCUSSION TITLE
New release ?

### DISCUSSION THREAD

**Discussion by Ludea** [iotaledger/stronghold.rs#318] [General] [answered]
With the Tauri rc release, do you plan to release a new version of tauri-plugin-stronghold and stronghold ?

**Comment by felsweg-iota** [comment, marked as answer]
Hey @Ludea ,
we are currently working on Stronghold towards a stable 1.0 release with an upcoming security audit to have everything in place. I presume the tauri-plugin-stronghold will be updated accordingly.

**Reply by Ludea** [reply]
So, still a few weeks of work ?

**Reply by felsweg-iota** [reply]
No ETAs :sweat_smile:  but we're good on track.

**Reply by Ludea** [reply]
Thanks for your answer !
Can't wait for it 😀

**Reply by felsweg-iota** [reply]
Thanks! We're also very excited about the upcoming stuff :blush:
